acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/condition/perm/GetNodesWhitelistPopulated.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl.condition.perm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.pantheon.tests.acceptance.dsl.condition.Condition;
+import tech.pegasys.pantheon.tests.acceptance.dsl.node.Node;
+import tech.pegasys.pantheon.tests.acceptance.dsl.transaction.perm.PermGetNodesWhitelistTransaction;
+
+import java.util.List;
+
+public class GetNodesWhitelistPopulated implements Condition {
+
+  private final PermGetNodesWhitelistTransaction transaction;
+
+  public GetNodesWhitelistPopulated(final PermGetNodesWhitelistTransaction transactions) {
+    this.transaction = transactions;
+  }
+
+  @Override
+  public void verify(final Node node) {
+    final List<String> response = node.execute(transaction);
+    assertThat(response).isInstanceOf(List.class);
+    assertThat(response).size().isEqualTo(3);
```

Comment:
Instead of having this hardcoded to 3 we should probably add a parameter to this condition with the expected list and use the size here.

Example: `AwaitNetPeerCount.java`